Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Location/UpdateLocation.hs

```diff
@@ -135,12 +136,15 @@ updateLocationHandler ::
 updateLocationHandler UpdateLocationHandle {..} waypoints = withLogTag "driverLocationUpdate" $
   withLogTag ("driverId-" <> driver.id.getId) $ do
     driverInfo <- DInfo.findById (cast driver.id) >>= fromMaybeM (PersonNotFound driver.id.getId)
+    transporterConfig <- CTC.findByMerchantId driverInfo.merchantId >>= fromMaybeM (TransporterConfigDoesNotExist driverInfo.merchantId.getId)
```

Comment:
hey, we already have merchantId in driver table (which is there in current scope also. `driver`). i dont think we need to add merchantId in driverInfo table as well.